CHANGELOG.md

```diff
@@ -6,7 +6,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Added
+
+- **CUMULUS-2576**
```

Comment:
2576 didn't go out in the 9.5.0 release?

Reply:
2576 is not in 9.5.0